sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -1099,6 +1106,7 @@ func resourceToMonitorsLibraryMonitor(d *schema.ResourceData) MonitorsLibraryMon
 		IsDisabled:         d.Get("is_disabled").(bool),
 		Status:             status,
 		GroupNotifications: d.Get("group_notifications").(bool),
+		Playbook:           d.Get("playbook").(string),
```

Comment:
please update the UTs to include this field (either in existing tests or new ones - but probably easier to add it to the existing tests)

Reply:
added acceptance test in ad992fe